docroot/modules/custom/va_gov_workflow/src/EventSubscriber/EntityEventSubscriber.php

```diff
@@ -2,7 +2,6 @@
 
 namespace Drupal\va_gov_workflow\EventSubscriber;
 
-use Drupal\block_content\BlockContentInterface;
 use Drupal\Core\Entity\ContentEntityForm;
```

Comment:
```suggestion
use Drupal\block_content\BlockContentInterface;
use Drupal\Core\Entity\ContentEntityForm;
```

See the change I suggested below that will require this to be added.